cmd/kubeadm/app/cmd/phases/kubeconfig_test.go

```diff
@@ -46,6 +46,8 @@ func TestKubeConfigCSubCommandsHasFlags(t *testing.T) {
 	cmdtestutil.AssertSubCommandHasFlags(t, []*cobra.Command{cmd}, "user", flags...)
 }
 
+const phaseTestK8sVersion = "v1.11.0"
```

Comment:
this seems to be a constant that we are passing around we probably should just clean it up later.
_test.go files depend on it.